packages/components/src/components/date-picker/date-picker-input.tsx

```diff
@@ -3,7 +3,7 @@ import classNames from 'classnames';
 import Icon from '../icon';
 import Input from '../input';
 
-const DatePickerIcon = ({ icon, ...props }: React.ComponentProps<typeof Icon>) => (
+const DatePickerIcon = ({ icon, ...props }:  React.ComponentProps<typeof Icon>) => (
```

Comment:
This file should not be in this PR. This is just the prettier change. Exclude it from the PR please.